packages/amplify_datastore/lib/types/DataStoreHubEvents/NetworkStatusEvent.dart

```diff
@@ -13,20 +13,11 @@
  * permissions and limitations under the License.
  */
 
-import 'package:flutter/services.dart';
+import 'package:amplify_core/types/hub/HubEventPayload.dart';
```

Comment:
amazing how git thinks its a fine rename when there is absolutely nothing in common except for top license header 🤷‍♂️